examples/llama2/llama2_tensor_parallel.json

```diff
@@ -34,6 +36,20 @@
                 "save_as_external_data": true,
                 "all_tensors_to_one_file": true
             }
+        },
+        "transformers_optimization_fp16": {
+            "type": "OrtTransformersOptimization",
+            "disable_search": true,

```

Comment:
nit: since `"search_strategy": false`, `disable_search` is redundant.

Reply:
Done!